### ISSUE TITLE
malformatted csv string

### ISSUE THREAD

**Issue by ccraddock** [FCP-INDI/C-PAC#735]
https://github.com/FCP-INDI/C-PAC/blob/547ae572146fe0744a8c363418eab5326aa5b945/CPAC/generate_motion_statistics/generate_motion_statistics.py#L890

This results in the malformed string:

Subject, Scan, MeanFD_Power, MeanFD_Jenkinson, NumFD_greater_than_0.50, rootMeanSquareFD, FDquartile(top1/4thFD), PercentFD_greater_than_0.50,MeanDVARSsub-M10943635_ses-NFB3, task-morald0.0527, 0.03050.0000, 0.17450.0578, 0.000018.6078

which is missing newline between header and line, as well as commas between many of the values

**Comment by ccraddock** [comment]
I have fixed this on the v1.0.2_disable_log  branch. But I think that we should develop a new pattern for performing these operations, i.e. calculating statistics across a variety of input files and then writing out a consolidated table. It would perhaps be less error prone if each stat was entered into a table at calculation, with a key indicating the correct column name. Then the output could be written as json, or could csv, the latter could be accomplished with pandas to insure correct formatting.

**Comment by sgiavasis** [comment]
@ccraddock Good idea- going to leave this issue open until I create a new one for that potential feature.

In the meantime, I've pulled in your branch changes into this 1.0.2a hotfix branch:
https://github.com/FCP-INDI/C-PAC/tree/1.0.2a_hotfix_dev

I'm going to run a full pipeline with it and re-run correlations to make sure everything looks good.

**Comment by nx10** [comment]
Outdated
